akka-http2-support/src/main/scala/akka/http/impl/engine/http2/FrameParser.scala

```diff
@@ -66,7 +65,7 @@ class FrameParser(shouldReadPreface: Boolean) extends ByteStringParser[FrameEven
           if (priority) payload.readByte() & 0xff
           else 0
 
-        // TODO: check that streamId != 0
+        Http2Compliance.requirePositiveStreamId(streamId)
```

Comment:
Hm, then we have this check in too many places right now... I'll remove it for now so we can merge and add those later